torch/csrc/api/include/torch/nn/modules/container/modulelist.h

```diff
@@ -89,7 +89,7 @@ class ModuleListImpl : public Cloneable<ModuleListImpl> {
   void push_back(std::shared_ptr<Module> module) {
     modules_.push_back(std::move(module));
     const auto index = modules_.size() - 1;
-    register_module(std::to_string(index), modules_[index]);
+    register_module(c10::to_string(index), modules_[index]);
```

Comment:
Curious are the `std::to_string` changes meant to be part of another PR https://github.com/pytorch/pytorch/pull/29839 ?

Reply:
Yeah, I think that happened because it's a stacked diff in fbsource.